tracer/src/Datadog.Trace/Processors/NormalizerTraceProcessor.cs

```diff
@@ -52,9 +52,12 @@ public ArraySegment<Span> Process(ArraySegment<Span> trace)
                 +----------+--------------------------------------------------------------------------------------------------------+
              */
 
-            for (var i = trace.Offset; i < trace.Count + trace.Offset; i++)
```

Comment:
Am I missing something or you removed all normalization here? You used to call `Process` on every span. And process did normalize many things. Should you keep the loop and just add the extra code for handling environment?

Reply:
Oops! Thank you. Fixed.